licenses/BSL.txt

```diff
@@ -7,9 +7,15 @@ Licensor:             Rocicorp, LLC
 Licensed Work:        repc 0.18.0
                       The Licensed Work is (c) 2020 Rocicorp, LLC.
 
-Additional Use Grant: You may use the Licensed Work when your application uses
-                      the Licensed Work with a total of less than 100 client
-                      instances for any purpose.
+Additional Use Grant: You may make use of the Licensed Work, provided you
+                      do not use it in any of the following ways:
+
+                      * Create a synchronization engine. A "synchronization
+                     engine" is a commercial or non-commercial offering that
+                     synchronizes data between two or more computers.
```

Comment:
Not sure if this is clear enough but I'm not sure how to improve it.

Maybe we can say something like "no derivative work" -- https://creativecommons.org/characteristic/nd?lang=en